src/translations/en.json

```diff
@@ -2037,6 +2037,9 @@
         "form": {
           "working": "Please wait",
           "unknown_error": "Something went wrong",
+          "next": "Next",
+          "start_over": "Start over",
+          "error": "Error:",
```

Comment:
Keep the `:` out of the translation.

Reply:
I think I did it wrong. It should be so `"error": "Error: {ErrorMessage}",`. What do you think?

Reply:
That's also possible although not really needed here as I think the order will stay the same in all languages.

Reply:
Do you want me to change this? The PR can be merged?